spec/authorization/index.md

```diff
@@ -45,6 +45,10 @@ registered patients and synchronizes these with an external database
 * A utilization tracking system that queries an EHR every minute for
 bed and room usage and displays statistics on a wall monitor.
 
+### Additional use Cases
+
+* Public health surveillance studies that do not require real-time exchange of data.
```

Comment:
There's one other Gforge issue asking for another use case to be added -- provider directories I think. Would be good if add that's issue's resolution here too.

Reply:
Yup thats been addressed here https://github.com/HL7/bulk-data/pull/7/commits/655713a5bca4a055436a57acda3e55202a34582f#diff-213427d3e24797581154795a8bb3ab22R30